.circleci/config.yml

```diff
@@ -173,8 +173,11 @@ jobs:
 
                       if [ $CIRCLE_BRANCH == $MASTER_BRANCH ]; then
                         git clone $MASTER_TARGET out
-                      else
+                      elif [ $CIRCLE_BRANCH == $DEV_BRANCH ]; then
                         git clone $DEV_TARGET out
+                      else
+                        yarn run deploy:feature
```

Comment:
Can we add an echo statement in here to say something like:
"Deploying feature branch to firebase: (url)"

Reply:
Yep you got it!